src/Nethermind/Nethermind.Hive.ConsensusWorkflowGenerator/Program.cs

```diff
@@ -0,0 +1,224 @@
+using System.Text;
+
+namespace Nethermind.Hive.ConsensusWorkflowGenerator;
+
+public static class Program
+{
+    const long MaxSizeWithoutSplitting = 35_000_000;
+    const long TargetSize = 23_000_000;
+    const long PenaltyForAdditionalInit = 300_000;
+
+    static void Main(string[] args)
+    {
+        StringBuilder fileContent = new();
+
+        List<string> directories = GetTestsDirectories();
+        Dictionary<string, long> pathsToBeTested = GetPathsToBeTested(directories);
+        List<List<string>> accumulatedJobs = GetTestsSplittedToJobs(pathsToBeTested);
+
+        WriteInitialLines(fileContent);
+
+        int jobsCreated = 0;
+        foreach (List<string> job in accumulatedJobs)
+        {
+            WriteJob(fileContent, job, ++jobsCreated);
```

Comment:
Better to write directly to file, which should be opened here and passed as TextWriter, instead of to StringBuilder